src/org/zaproxy/zap/extension/ascanrules/BufferOverflow.java

```diff
@@ -62,7 +66,12 @@ public String getName() {
 	public String[] getDependency() {
 		return null;
 	}
-
+	
+	@Override
+	public boolean targets(TechSet technologies) { 

```

Comment:
This should be mentioned in the commit message (or done in another commit).
I guess this should be mentioned in the help pages too.